import/import_script.go

```diff
@@ -0,0 +1,897 @@
+	if clustersResp.JSON200 == nil {
+		return "", fmt.Errorf("failed to list clusters, JSON200 resp is nil, organizationId: %v", organizationId)
+	}
+
+	_, diagnostic := clients.NormalizeAPIError(ctx, clustersResp.HTTPResponse, clustersResp.Body)
+	if diagnostic != nil {
+		log.Printf("API Error diagnostic: %+v", diagnostic)
+	}
+
+	clusters := clustersResp.JSON200.Clusters
+	if clusters == nil {
+		return "", fmt.Errorf("clusters list is nil")
+	}
+
+	clusterMap := make(map[string]platform.ClusterType)
+	for _, cluster := range clusters {
+		if cluster.Id != "" {
+			clusterMap[cluster.Id] = cluster.Type
+		}
+	}
+
+	log.Printf("Importing Clusters: %v", maps.Keys(clusterMap))
+
+	var importString string
+	var clusterImportString string
+	for clusterId, clusterType := range clusterMap {
+		if clusterType != platform.ClusterTypeHYBRID {
+			clusterImportString = fmt.Sprintf(`
+import {
+	id = "%v"
+	to = astro_cluster.cluster_%v
+}`, clusterId, clusterId)
+		}
+
+		if clusterType == platform.ClusterTypeHYBRID {
+			clusterImportString += fmt.Sprintf(`
+import {
+	id = "%v"
+	to = astro_hybrid_cluster_workspace_authorization.cluster_%v
+}`, clusterId, clusterId)
+		}
+
+		importString += clusterImportString + "\n"
+	}
+
+	return importString, nil
+}
+
+func handleApiTokens(ctx context.Context, platformClient *platform.ClientWithResponses, iamClient *iam.ClientWithResponses, organizationId string) (string, error) {
+	log.Printf("Importing API tokens for organization %s", organizationId)
+
+	apiTokensResp, err := iamClient.ListApiTokensWithResponse(ctx, organizationId, nil)
+	if err != nil {
+		return "", fmt.Errorf("failed to list API tokens: %v", err)
+	}
+
+	if apiTokensResp.StatusCode() != http.StatusOK {
+		return "", fmt.Errorf("unexpected status code: %d, body: %s", apiTokensResp.StatusCode(), string(apiTokensResp.Body))
+	}
+
+	if apiTokensResp.JSON200 == nil {
+		return "", fmt.Errorf("failed to list API tokens, JSON200 resp is nil, organizationId: %v", organizationId)
+	}
+
+	_, diagnostic := clients.NormalizeAPIError(ctx, apiTokensResp.HTTPResponse, apiTokensResp.Body)
+	if diagnostic != nil {
+		log.Printf("API Error diagnostic: %+v", diagnostic)
+	}
+
+	apiTokens := apiTokensResp.JSON200.Tokens
+	if apiTokens == nil {
+		return "", fmt.Errorf("API tokens list is nil")
+	}
+
+	apiTokenIds := lo.Map(apiTokens, func(apiToken iam.ApiToken, _ int) string {
+		return apiToken.Id
+	})
+
+	log.Printf("Importing API Tokens: %v", apiTokenIds)
+
+	var importString string
+	for _, apiTokenId := range apiTokenIds {
+		apiTokenImportString := fmt.Sprintf(`
+import {
+	id = "%v"
+	to = astro_api_token.api_token_%v
+}`, apiTokenId, apiTokenId)
+
+		importString += apiTokenImportString + "\n"
+	}
+
+	return importString, nil
+}
+
+func handleTeams(ctx context.Context, platformClient *platform.ClientWithResponses, iamClient *iam.ClientWithResponses, organizationId string) (string, error) {
+	log.Printf("Importing teams for organization %s", organizationId)
```

Comment:
do we need to worry about scim for these

Reply:
yes we should exit if scim